nightly/nightly.txt

```diff
@@ -117,6 +122,7 @@ expensive nearcore test_cases_testnet_rpc test::test_add_access_key_with_allowan
 expensive nearcore test_cases_testnet_rpc test::test_delete_access_key_with_allowance_testnet
 # The next test is disabled due to #2208
 # expensive nearcore test_cases_testnet_rpc test::test_access_key_smart_contract_testnet
+expensive nearcore test_cases_testnet_rpc test::test_access_key_smart_contract_testnet
```

Comment:
Probably should remove the previous comment then?

Reply:
Probably should make these as "very_expensive_tests" feature, so
- expensive tests must be in nightly tests and run, and this is checked on ci
- very expensive tests are not

Reply:
Oh comment the test also works, with this approach "very expensive tests" should be commented in rust.

Reply:
I think you are commenting the wrong one.
The very expensive test is `test_all_chunks_accepted_1000_slow` (though I would actually check how long it takes, it might be acceptable to have it in nightly).

With this test my concern is that we have the test included immediately following a comment that says that the test is *not* included.

Reply:
I see. Yep. So should this one be enabled?
> # expensive nearcore test_cases_testnet_rpc test::test_access_key_smart_contract_testnet

Reply:
Let's disccuss it here: #2758

Effectively the question is: do we want to have tests that are known to be failing included in nightly.